frontend/internal-packages/schema-bench/benchmark-workspace-default/execution/input/case-001-difficult.json

```diff
@@ -1,3 +1,3 @@
project, the total required quantity is recorded.\n     * Although Company A usually procures the items, some items are provided free of charge by the client. The quantity of such items is recorded as the client-provided quantity.\n     * The actual required procurement quantity is the total required quantity minus the client-provided quantity.\n\n### 7. From Request for Quotation to Receipt of Quotation\n\n1. When procuring items, Company A sends RFQs (Requests for Quotation) to multiple suppliers that handle the brand of the item.\n\n   * Each RFQ is given a request number and the request date is recorded. It must be linked to the corresponding prototype project.\n   * The client’s name cannot be disclosed to suppliers, but to get advice from suppliers in case the client selects an inappropriate item, the product purpose is shared.\n   * Each item in the RFQ is assigned an RFQ detail number, along with the required quantity and desired delivery date.\n   * Suppliers are requested to include the RFQ number and RFQ detail number in their quotation responses.\n2. Quotations are received from suppliers. A single quotation is expected per RFQ.\n\n   * Each quotation includes the RFQ number, validity period, quotation date, and a supplier-issued quotation number (which may not be unique across suppliers).\n   * Each quotation detail includes the RFQ detail number, manufacturer part number, procurement conditions, and a supplier-issued quotation detail number (which may not be unique across suppliers). Some items may not receive a quotation.\n   * A quotation detail may propose different items than those requested. In such cases, the reason for the proposal is recorded.\n   * A single item may have multiple procurement conditions in a quotation detail. For example, if the required quantity is 300 units and the lot size is 1,000, the supplier may quote:\n\n     * ¥0.5 per unit for a lot size of 1,000 units\n     * ¥2 per unit for a lot size of 1 unit\n\n### 8. From Purchase Order to Receiving\n\n1. After consulting with the client and reviewing the quotations, one appropriate procurement condition is selected for each item.\n\n   * The selected quotation detail is recorded in the purchase order detail, along with the number of lots to order and the specified delivery date.\n   * Order details issued to the same supplier around the same time are grouped into a single purchase order, even across different prototype projects.\n   * Each purchase order is assigned a purchase order number, along with the order date and total order amount.\n2. Items are received from suppliers based on the purchase order.\n\n   * Each receipt is assigned a receipt number and the receipt date is recorded.\n   * Each item received is assigned a receipt detail number. A purchase order detail corresponds to a single receipt (no split deliveries).\n   * A label with the receipt number and detail number is attached to the item packaging and handed over to the production team.\n\n"
```

Comment:
It’s not perfect, but I don’t think this will make things worse, so let me give it a try.










- before: Perform conceptual data modeling...
- after: design the schema: ...